src/parsing/test/testing.ts

```diff
@@ -293,6 +304,30 @@ export function toEqualDefinition(actualResponse, testDefinition) {
         testDefinition.expectedResult.lowerCase = false;
     }
 
+    if (testDefinition.noErrors === false && testDefinition.expectedErrors) {
+        const filteredResponseErrors = actualResponse.errors.map((responseError, index) => {
```

Comment:
We filtering first level keys, imo its enough for current interface and this code part did his job perfectly

Reply:
What if we don't have errors? Please handle this case